tests/e2e/test_spyre_cb_scheduler_steps.py

```diff
@@ -686,8 +686,8 @@ def test_prompt_too_long_for_current_tkv(model: ModelInfo, backend: str,
             * 1: len = 70, max tokens = 4, step joining = 0
     """
 
-    if prefill_optimization:
-        monkeypatch.setenv('VLLM_SPYRE_ENABLE_PREFILL_OPTIMIZATION', '1')
+    if not prefill_optimization:
+        monkeypatch.setenv('VLLM_SPYRE_ENABLE_PREFILL_OPTIMIZATION', '0')
```

Comment:
Maybe it would be better to not assume any default value and set it anyway. It makes things more explicit.

```suggestion
    monkeypatch.setenv('VLLM_SPYRE_ENABLE_PREFILL_OPTIMIZATION', '1' if prefill_optimization else '0')
```